modules/oc-mirror-workflows-table.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-workflows-table_{context}"]
+= oc-mirror comparison table
+
+[cols="1,1,1,1,1",options="header"]
+|===
+| Criteria
+| Mirror-to-disk (m2d)
+| Disk-to-mirror (d2m)
+| Mirror-to-mirror (m2m)
+
+| When to use
+| Run on a machine with internet access to pull and package content.
+| Run inside an air-gapped environment to load content into the target registry.
+| Run on a machine that has internet access and network connectivity to the target registry.
```

Comment:
For disk-to-mirror you don't need internet connection , only a connection to the remote-registry